CHANGELOG.md

```diff
@@ -0,0 +1,57 @@
+# Changelog
+
+All notable changes to the Carbon Aware SDK will be documented in this file.
+
+## [Unreleased]
+
+### Added 
+
+- Added Electricity Map support for forecasting.  This supports the commercial Electricity Map API only, and currently does not support the free Electrity Map API due to differences in the API's.
+
+### Fixed
+
+- Fixed generated test data that had time bomb bug that was in test data, which caused integration tests to fail.  This is now automatically generated each time.
+
+### Changed
+
+#### API
+
+- No API changes have been made
+
+#### API Deployment
+
+- Configuration has changed.  Refer to upgrading from 1.0.0 to 1.1.0 below.
+
+#### SDK 
+
+- Funcationality for forecast and historical data have been seperated into seperate interfaces.  This impacts configuration, see upgrading from 1.0.0 to 1.1.0 for more information
+- Additional tests
+
+### Upgrading from 1.0.0 to 1.1.0 
+
+- Configuration changes are required due to historical and forecast configuration now being decoupled.  Refer to ___ for a guide.
```

Comment:
There is no guide yet for upgrading to v1.1 from v1.0

Reply:
Correct, because we don't have 1.1 yet.  This is just for the changelog, we need the guide to be built.